TIKEBook_Lesson1.ipynb

```diff
@@ -0,0 +1,1247 @@
+{
```

Comment:
<p>Maybe add a more explicit definition/explanation of HDUlists and why they are important/used, rather than making the reader go back and figure it out based off of the return</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
Lesson 0 has an in-depth explanation on FITS and HDUlists and I have been told to not repeat large parts of previous lessons for redundancy. I will need further go-ahead on the next review to add it to this one.